app/models/katello/repository.rb

```diff
@@ -483,6 +488,39 @@ def import_system_applicability
       end
     end
 
+    def self.import_distributions
+      self.all.each do |repo|
+        repo.import_distribution_data
+      end
+    end
+
+    def import_distribution_data
+      distribution = Katello.pulp_server.extensions.repository.distributions(self.pulp_id)
+      if distribution.any?
+        distribution = distribution[0]
+        self.update_attributes!(:distribution_version => distribution["version"],
+                                :distribution_arch => distribution["arch"],
+                                :distribution_family => distribution["family"],
+                                :distribution_variant => distribution["variant"],
+                                :distribution_bootable => distribution_bootable?(distribution)
+                                )
+      end
+    end
+
+    def distribution_bootable?(distribution = nil)
```

Comment:
This could probably be a class method in glue/pulp/repo.rb that takes in the distribution object, something like:  Repository.distribution_bootable?(distribution)

That might help differentiate between the two.